requirements/docs.txt

```diff
@@ -381,16 +370,14 @@ charset-normalizer==3.3.2 \
     --hash=sha256:fb69256e180cb6c8a894fee62b3afebae785babc1ee98b81cdf68bbca1987f33 \
     --hash=sha256:fd1abc0d89e30cc4e02e4064dc67fcc51bd941eb395c502aac3ec19fab46b519 \
     --hash=sha256:ff8fa367d09b717b2a17a052544193ad76cd49979c805768879cb63d9ca50561
-    # via
-    #   aiohttp
-    #   requests
+    # via requests
 click==8.1.7 \
     --hash=sha256:ae74fb96c20a0277a1d615f1e4d73c8414f5a98db8b799a7931d1582f3390c28 \
     --hash=sha256:ca9853ad459e787e2192211578cc907e7594e294c7ccc834310722b41b9ca6de
     # via flask
-connexion==3.0.1 \
-    --hash=sha256:b8a4d07ceab452d5033bdf0ebf3489a19217abbdfb0b0b02eddf1d5da2d7e997 \
-    --hash=sha256:df1edd47fad7fc6c8571845a274ba9571e8b1ed96258ff98dd2ffd713517672a
+connexion==3.0.2 \
```

Comment:
Should we also pin connexion to <3 in the docs build for consistency with base.in?